temp.py

```diff
@@ -0,0 +1,8 @@
+def reset_files(filepath, file_type="txt"):
+    with open(filepath, "w") as output_file:
```

Comment:
Consider adding a check to ensure the file exists before trying to open it. This can prevent runtime errors. [important]